### PR TITLE
Fixes #4872 fix(tabledemo): fixes removal of bullseye layouts

### LINKED ISSUE #4872: Table demo cleanup
Placeholder for the demo work being done in react <- done!

* remove bullseye layout from loading and empty state demos

### PR DESCRIPTION
**Pull request by blfetche**
Removed reference to bullseye handle bar code

### CODE REVIEW

File: src/patternfly/demos/Table/table-empty-state-table.hbs [hunk 1]
```diff
@@ -27,11 +27,10 @@
   {{#> table-tbody}}
     {{#> table-tr}}
       {{#> table-td table-td--attribute='colspan="8"'}}
-        {{#> bullseye}}
           {{#> empty-state empty-state--modifier="pf-m-sm"}}
             {{#> empty-state-icon empty-state-icon--type="search"}}{{/empty-state-icon}}
             {{#> title titleType="h2" title--modifier="pf-m-lg"}}
-              No results found
+              No results found Testing
```
Comment (srambach) [inline src/patternfly/demos/Table/table-empty-state-table.hbs:33]: You'll want to remove this 😉

**Review by srambach — COMMENTED** [review]
Just one tiny thing, otherwise great! Welcome to handlebars 🤣

File: src/patternfly/demos/Table/table-empty-state-table.hbs (hunk 1 above)
Comment (blfetche) [reply src/patternfly/demos/Table/table-empty-state-table.hbs]: Corrected and thank you.

**Review by srambach — APPROVED** [review]
Looks perfect! 🎉

**Review by mattnolting — APPROVED** [review]
LGTM! 👍

**Comment by blfetche** [comment]
Thanks all!

**Pull request merged**